pkg/base1/test-external.js

```diff
@@ -26,6 +26,38 @@ QUnit.test("external get", function (assert) {
     req.send();
 });
 
+QUnit.test("external fsread1", async assert => {
+    const done = assert.async();
+    assert.expect(5);
+    const resp = await cockpit.spawn(["bash", "-c", "size=$(stat --format '%s' /usr/lib/os-release); echo $size"]);
```

Comment:
Was this seriously the easiest way to figure this out?  Ugh.

Reply:
The test runs in the browser so we have to use cockpit somehow :)

Reply:
@allisonkarlitskaya probably means that you should just do `spawn(["stat", "--format", "%s", "/usr/lib/os-release"])` and drop the shell wrapping :grin:

Reply:
ooo good point!